core/src/main/java/com/linecorp/armeria/server/HttpResponseSubscriber.java

```diff
@@ -53,10 +53,15 @@ void onResponseHeaders(ResponseHeaders headers) {
         } else {
             if (req.method() == HttpMethod.HEAD) {
                 endOfStream = true;
```

Comment:
Question) maybe nit, but even if the method is HEAD couldn't trailers be sent also?

```java
endOfStream = reqCtx.additionalResponseTrailers().isEmpty();
```

Reply:
I believe we can ignore response trailers because the clients do not expect to see trailers from HEAD requests. https://http.dev/head